src/Account/AccountCollection.php

```diff
@@ -7,7 +7,7 @@
 /**
  * @author Milan Matějček
  */
-class Accounts
+class AccountCollection implements \Countable, \IteratorAggregate
```

Comment:
Foreach maybe but count of account? I think you don't need. Why you don't use inteface \Iterator?